src/components/LineChart.vue

```diff
@@ -1,85 +0,0 @@
-<template>
-  <div>
-    <apexcharts
-      :width="width"
-      :height="height"
-      type="line"
-      :options="chartOptions"
-      :series="series"
-    />
-  </div>
-</template>
-
-<script>
-import VueApexCharts from 'vue3-apexcharts'
-import { timeConverter } from '../fetchStats'
-
-export default {
-  name: 'LineChart',
-  components: {
-    apexcharts: VueApexCharts
-  },
-  props: {
-    title: String,
-    times: Array,
-    fromMeToGmail: Array,
-    width: String || Number,
-    height: String || Number
-  },
-  data: function () {
-    return {
```

Comment:
For constants, you can use `computed` instead. Variables in `data` are mutable. `computed` is not.

Reply:
Gotcha. This resolves a question I asked earlier. Thanks!